cc_licenses/templates/includes/legalcode_40_license.html

```diff
@@ -0,0 +1,181 @@
+{% load cctrans %}
+<div id="legal-code-body" class="padding-larger margin-top-bigger has-text-black">
```

Comment:
Since this is a new template. We need to add `{% load bidi %}` and update where there is `padding-left-*`/`padding-right-*` to make use of `{% start %}`/`{% end %}`.

Reply:
Will do.